1.7/usage/managing-services/uninstall.md

```diff
@@ -12,7 +12,7 @@ Services can be uninstalled from either the web interface or the CLI. If the ser
 
 ### Uninstalling a service using the CLI
 
-1.  Uninstall a datacenter service with this command:
+Uninstall a datacenter service with this command:
 
     ```bash
```

Comment:
You need to remove the command indentation since you have deleted the number, otherwise it renders odd: https://cl.ly/2C0l1R3P2z3N